brave/brave-core
# Clear display coordinate pref if a custom notification wasn’t dragged.

Resolves https://github.com/brave/brave-browser/issues/33747



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

* Run the browser with the previous version
* Trigger custom notification ad
* Move the custom ad notification from the default position
* Install the new version of the browser
* Trigger custom notification ad
* Make sure that the notification position didn't change compared to previous run
